src/js/utils/log.js

```diff
@@ -62,7 +67,7 @@ export const logByType = (type, args, stringify = !!IE_VERSION && IE_VERSION < 1
   if (!fn.apply) {
     fn(args);
   } else {
-    fn[Array.isArray(args) ? 'apply' : 'call'](console, args);
+    fn[Array.isArray(args) ? 'apply' : 'call'](window.console, args);
```

Comment:
Yeah, the implied global `console` combined with the noop (which had `apply()`) was causing the problem in the first place.

As to the other change, I think I want to keep this fix pretty minimal. More changes are more opportunities for bugs!